### DISCUSSION TITLE
dahboard-08

### DISCUSSION THREAD

**Discussion by euotiniel** [shadcn-ui/ui#3408] [Blocks Request]
### Description

A simple and minimalist version of a dashboard. Developed by me, using shadcn.ui

![Captura de tela_7-4-2024_115043_localhost](https://github.com/shadcn-ui/ui/assets/93485038/8421eae7-fc63-4ea2-92fc-06434a8126df)

**Comment by ArielMejiaDev** [comment]
@euotiniel did you have in mind to make a PR about it? looks nice

**Reply by euotiniel** [reply]
@ArielMejiaDev I don't know if it would need to be validated first. Are you part of the team?

**Reply by ArielMejiaDev** [reply]
@euotiniel no I am not part of the team, just asking, the layout that you propose looks pretty generic and useful

**Comment by euotiniel** [comment]
@rdrigos and @alvinsjoy, Your opinion on this block?

**Reply by alvinsjoy** [reply]
Looks great. I'm not a part of the team either, maybe try mentioning shadcn.

**Reply by rdrigos** [reply]
I agree with @alvinsjoy , the layout is intuitive, beautiful and consistent. But as I'm not part of the team, it would be ideal to mention @shadcn

**Comment by euotiniel** [comment]
I still hope to send a PR. @shadcn
